packages/manager/modules/pci/src/projects/new/module.js

```diff
@@ -12,7 +12,7 @@ import routing from './routing';
 import component from './component';
 import service from './service';
 
-import orderCart from './order-cart.service'; // TODO remove it
+import orderCart from '../order-cart.service'; // TODO remove it
```

Comment:
TODO ?

Reply:
@frenautvh this sentence was here when i started.
I let it until @Jisay be back to ask him why he wrote this sentence.

Reply:
Hello @frenautvh, so i discussed with @Jisay about the comment.
He said is temporary copy of the same file as here:
```packages/manager/modules/web-universe-components/src/order-cart/order-cart.service.js``` 

So i discussed with @JeremyDec and @cbourgois. So, i planned to make a refactor an another task.
Thanks for comment.